### DISCUSSION TITLE
[gatsby-theme-minimal-blog]: Changing font family

### DISCUSSION THREAD

**Discussion by alexandra-lim** [LekoArts/gatsby-themes#639] [Questions] [answered]
I've been trying to change the font family of the minimal blog theme by updating the values in the `gatsby-omni-font-loader` config with no luck. I'm following the example as shown in the README of the plugin, but the font family is defaulting to one of IBM Plex's fallbacks.

Am I supposed to be shadowing the `layout.tsx` file? How would I make this change?

**Comment by LekoArts** [comment, marked as answer]
Hi!

I've added a section to the README to explain it :)

https://github.com/LekoArts/gatsby-themes/tree/master/examples/minimal-blog#changing-your-fonts

**Reply by alexandra-lim** [reply]
Thank you so much! Really appreciate this!
